editor/property_selector.cpp

```diff
@@ -554,27 +572,35 @@ void PropertySelector::set_type_filter(const Vector<Variant::Type> &p_type_filte
 	type_filter = p_type_filter;
 }
 
+void PropertySelector::set_multiselect(bool p_is_multiselect) {
+	search_options->set_select_mode(p_is_multiselect ? search_options->SELECT_MULTI : search_options->SELECT_SINGLE);
+}
+
 void PropertySelector::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("selected", PropertyInfo(Variant::STRING, "name")));
+	ADD_SIGNAL(MethodInfo("multiple_properties_selected", PropertyInfo(Variant::ARRAY, "name")));
 }
 
 PropertySelector::PropertySelector() {
 	VBoxContainer *vbc = memnew(VBoxContainer);
 	add_child(vbc);
 	//set_child_rect(vbc);
+
 	search_box = memnew(LineEdit);
 	vbc->add_margin_child(TTR("Search:"), search_box);
 	search_box->connect(SceneStringName(text_changed), callable_mp(this, &PropertySelector::_text_changed));
 	search_box->connect(SceneStringName(gui_input), callable_mp(this, &PropertySelector::_sbox_input));
 	search_options = memnew(Tree);
 	search_options->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
+	search_options->set_select_mode(search_options->SELECT_SINGLE);
 	vbc->add_margin_child(TTR("Matches:"), search_options, true);
-	set_ok_button_text(TTR("Open"));
+	set_ok_button_text(TTR("Select"));
```

Comment:
I think Select is more appropriate for the property selector, as we aren't `Open` ing anything :thinking: